src/main/java/seedu/duke/commands/Command.java

```diff
@@ -0,0 +1,21 @@
+package seedu.duke.commands;
+
+import seedu.duke.storage.Storage;
+import seedu.duke.storage.TaskList;
+
+public abstract class Command {
+    /**
+     * Helps to execute the command.
+     * 
+     * @param taskList the list of Tasks which is being stored.
+     * @param storage  the database where the Tasks are being saved for progression.
+     */
+    public abstract void execute(TaskList taskList, Storage storage);
+
+    /**
+     * Checks if the user wants to exit from the application.
+     * 
+     * @return boolean whether the user wants to exit from the application.
+     */
+    public abstract boolean getIsExit();
```

Comment:
I understand that it is a getter method, but I think it might be better to name the method `isExit` instead.

Reply:
Second this. Especially due to the JavaDocs description. Based on implementations on other classes it doesn't seem to be a getter method rather a check method.